FirebaseSwiftUI/FirebaseGoogleSwiftUI/Sources/Services/GoogleProviderAuthUI.swift

```diff
@@ -20,23 +20,19 @@ public class GoogleProviderAuthUI: @preconcurrency GoogleProviderAuthUIProtocol
   let scopes: [String]
   let shortName = "Google"
   let providerId = "google.com"
-  let clientID: String
+  public let clientID: String
   public init(scopes: [String]? = nil, clientID: String = FirebaseApp.app()!.options.clientID!) {
     self.scopes = scopes ?? kDefaultScopes
     self.clientID = clientID
   }
 
   @MainActor public func authButton() -> AnyView {
-    let customViewModel = GoogleSignInButtonViewModel(
-      scheme: .light,
-      style: .wide,
-      state: .normal
-    )
-    return AnyView(GoogleSignInButton(viewModel: customViewModel) {
-      Task {
-        try await self.signInWithGoogle(clientID: self.clientID)
-      }
-    })
+    AnyView(SignInWithGoogleButton())
```

Comment:
Moved this into separate View, it was directly calling signInWithGoogle which meant it wasn't actually signing in with Firebase and not using AuthService to update currentUser and other aspects of state.

Reply:
Consider documenting this in a comment since it's unintuitive looking at the code.